index.css

```diff
@@ -0,0 +1,93 @@
+*{
+    border-color: rgba(0,0,0,0);
+    border-style: solid;
+    margin: 0;
+    padding: 0p;
+}
+body{
+    display: block;
+    border-width: 0px 33vw;
+}
+h1{
+    display: inline-block;
+    font-weight: 900;
+    font-family: 'Merriweather', serif;
+    border-width: 30px 0px 0px 25px;
+    color: white;
+}
+p{
+    font-weight: 400;
+    font-family: 'Roboto', serif;
+    font-size: 13px;
+    color: white;
+    border-width: 15px 0px 30px 25px;
+}
+article{
+    background-size: cover;
+    border-width: 0px 0px 25px 0px;
+    border-color: silver;
+}
+.btnSection{
+    border-width: 0px 0px 10px 25px;
+}
+.parentBtn{
+    background: inherit;
+    position: relative;
+    margin-left: 3px;
+    margin-bottom: 10px;
+    display: inline-block;
+    height: 30px;
+    width: fit-content;
+    padding: 1px 10px;
+    border-radius: 25px;
+    overflow: hidden;
+}
+@keyframes btnTransition{
```

Comment:
Instead of `@keyframes` block, you could (example)

.button { color: red; transition: color 0.35s ease-in-out; }
.button:hover { color: red }

Same applies for any other property, or for more of them (then, instead of `color` write `any`)

Reply:
But hey, I am glad you did this anyway! 😄